tests/unit_tests/analysis/test_es_update.py

```diff
@@ -36,18 +36,25 @@ def remove_timestamp_from_logfile(log_file: Path):
         fout.write(buf)
 
 
-def test_update_report(
-    snake_oil_case_storage, snake_oil_storage, new_ensemble, snapshot
-):
+def test_update_report(snake_oil_case_storage, snake_oil_storage, snapshot):
     """
     Note that this is now a snapshot test, so there is no guarantee that the
     snapshots are correct, they are just documenting the current behavior.
     """
     ert = snake_oil_case_storage
     es_update = ESUpdate(ert)
+    prior_ens = snake_oil_storage.get_ensemble_by_name("default_0")
+    experiment = snake_oil_storage.get_experiment(prior_ens.experiment_id)
+    posterior_ens = snake_oil_storage.create_ensemble(
+        experiment,
```

Comment:
Think you can just do: prior_ens.experiment_id here

Reply:
Same for the other instances as well

Reply:
Fixed, thanks 👍